Presentation.Web/Controllers/API/V1/UI_Configuration/UIConfigurationController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web;
+using System.Web.Http;
+using Presentation.Web.Infrastructure.Attributes;
+using Presentation.Web.Models.API.V1;
+using Presentation.Web.Models.API.V1.UI_Configuration;
+using Swashbuckle.Swagger.Annotations;
+
+namespace Presentation.Web.Controllers.API.V1.UI_Configuration
+{
+    [InternalApi]
+    [RoutePrefix("api/v1/organizations/{organizationId}/ui-config/modules/{module}")]
+    public class UIConfigurationController : BaseApiController
```

Comment:
rename to UIModuleconfiguratioController since it is scoped to config of modules :-)